components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -40,9 +56,30 @@ class CustomTabsToolbarFeature(
         }
     }
 
+    internal fun addCloseButton(bitmap: Bitmap?) {
+        val drawableIcon = bitmap?.let { icon ->
+            BitmapDrawable(context.resources, icon).also {
+                if (it.isSmallerThan(MAX_CLOSE_BUTTON_SIZE_DP)) return@also
+            }
+        } ?: ContextCompat.getDrawable(context, R.drawable.mozac_ic_close)
+
+        drawableIcon?.let {
+            val button = Toolbar.ActionButton(
+                it, context.getString(R.string.mozac_feature_customtabs_exit_button)
+            ) { closeListener.invoke() }
+            toolbar.addNavigationAction(button)
+        }
```

Comment:
`drawableIcon?` confuses me here. Can we really end up in a situation where we do not have the icon and are not adding a button?

Reply:
We could because the bitmap is nullable, and if we use `ContextCompat.getDrawable` to get `mozac_ic_close`, it also has a nullable annotation on that function.